model-optimizer/extensions/front/caffe/mvn_ext.py

```diff
@@ -30,5 +30,5 @@ def extract(cls, node):
         attrs = collect_attributes(param)
 
         # update the attributes of the node
-        MVN.update_node_stat(node, attrs)
+        MVNCaffe.update_node_stat(node, attrs)
```

Comment:
Why do we need `MVNCaffe` class if `MVNNormalizer` is triggered anyway? I think this is redundant.

Reply:
Because MVNCaffe and MVN have different attributes and number of inputs.